src/routes/Home.js

```diff
@@ -1,11 +1,17 @@
 import React from 'react';
-import BookLists from '../components/BookLists';
-import AddBook from '../components/AddBook';
+import BookLists from '../components/BookItem';
+import AddBook from '../components/BookData';
 
 const Home = () => (
-  <div>
-    <BookLists />
-    <AddBook />
+  <div className="container">
+    <div className="store-container">
+      <BookLists />
+    </div>
+    <div className="Line" />
+    <div className="form-container">
+      <h2 className="form-title">ADD NEW BOOK</h2>
+      <AddBook />
+    </div>
```

Comment:
- [ ] Kindly put some spacing between the heading `ADD NEW BOOK` and the form below. Please use uppercase letters for the text inside the button `ADD BOOK` 🤝️

**Your design**
![Screenshot from 2023-04-05 15-17-44](https://user-images.githubusercontent.com/17243325/230083402-bc48888b-190e-44f7-951f-48d6413c8bf0.png)

**Template**
![Screenshot from 2023-04-05 15-17-24](https://user-images.githubusercontent.com/17243325/230083397-aefc5d47-04da-42e9-bc4b-3efb5d635cc5.png)